tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -336,8 +353,18 @@
               "min_needed": null,
               "explanation": null,
               "course_list": [
-                "CEE 306: Hydrology",
-                "CEE 307: Field Ecohydrology"
+                "CEE 205: Mechanics of Solids",
+                "MAE 223 Modern Solid Mechanics"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 302: Practical Models for Environmental Systems",
```

Comment:
"CEE 302 Practical Models for Environmental Systems or CEE 304 Environmental Engineering and Energy"